pkg/pfconfig/shared/shared.go

```diff
@@ -209,8 +209,8 @@ type RFConfig struct {
 	Enable      bool    `json:"enable"`
 	Type        string  `json:"type,omitempty"`
 	Frequency   uint64  `json:"freq"`
-	RSSIOffset  float32 `json:"rssi_offset"`
-	TxEnable    bool    `json:"tx_enable"`
+	RSSIOffset  float32 `json:"rssi_offset,omitempty"`
+	TxEnable    bool    `json:"tx_enable,omitempty"`
```

Comment:
You need to make these pointers so that we can make a distinction between the zero value and "omit"